### PR TITLE
Merge staging-a into protocol-delivery

### PR DESCRIPTION
**Pull request by IanLondon**
updates the module parsing stuff, and bumps OT2 monorepo version

### CODE REVIEW

File: protolib2/parse/parseOT2.py [hunk 1]
```diff
@@ -116,12 +116,11 @@ def Pipette(self, *args, **kwargs):
         robot, protocol_function, all_labware)
 
 
-# TODO: Add in all recorded modules to the result dict once deck map can
-# support it.
 def get_result_dict(robot, protocol_function, all_labware):
```
Comment (Laura-Danielle) [inline protolib2/parse/parseOT2.py]: You don't need to add the modules to the signature?

File: protolib2/parse/parseOT2.py [hunk 2]
```diff
@@ -18,20 +18,23 @@
...
+            'name': label or labware_name,
+            'share': share
+            })
     return orig_labware_load(labware_name, slot, label, share)
 
 
-# Create fake loading function for modules
+# modules.load spy
 def load_module_spy(module_name, slot):
+    print('debug module', module_name, slot)
```
Comment (Laura-Danielle) [inline protolib2/parse/parseOT2.py]: Do you want this print statement here?

File: protolib2/parse/parseOT2.py [hunk 3]
```diff
@@ -41,6 +44,8 @@ def load_module_spy(module_name, slot):
 # monkeypatch the spies in
 labware.load = load_labware_spy
 modules.load = load_module_spy
+# TODO: Ian 2018-09-13 avoid these spies altogether once there's a solid way
+# to get labware and modules (eg after major Session refactor)
```
Comment (Laura-Danielle) [inline protolib2/parse/parseOT2.py]: 😿

File: protolib2/parse/parseOT2.py (hunk 2 above)
Comment (IanLondon) [reply protolib2/parse/parseOT2.py]: nope d'oh

**Pull request merged**